src/org/traccar/database/NotificationManager.java

```diff
@@ -62,6 +66,9 @@ public void updateEvent(Event event, Position position) {
                 if (notification != null) {
                     if (notification.getWeb()) {
                         Context.getConnectionManager().updateEvent(userId, event);
+                        if (withPosition) {
+                            Context.getConnectionManager().updatePosition(userId, position);
```

Comment:
Because related position from past we need resent it after event to correctly link in web-client in "Events panel"